pytransform3d/visualizer.py

```diff
@@ -74,6 +74,16 @@ def add_geometry(self, geometry):
                 Open3D geometry.
             """
             self.visualizer.add_geometry(geometry)
+            
+        def remove_geometry(self, geometry):
```

Comment:
Does this need to be public? In future versions of pytransform3d we might use the new interface to the Open3D visualizer, which might require to break this interface.

Here is an example:

https://github.com/intel-isl/Open3D/blob/b667a5565cbdedf6c97d356fb19bdb020d111bb0/examples/python/gui/add-geometry.py#L98